fivox/uriHandler.cpp

```diff
@@ -484,6 +484,57 @@ std::string URIHandler::getDescription() const
     return _impl->getDescription();
 }
 
+std::string URIHandler::getHelp()
+{
+    return
+//! [VolumeParameters] @anchor VolumeParameters
```

Comment:
How does the snippet anchor thingy work?
My guess is that lines are simply copied from here to the output, which means that the first and last lines are going to look bad.